### ISSUE TITLE
1.2.1 fails to load into makecode

### ISSUE THREAD

**Issue by nsteinmetz** [DFRobot/pxt-maqueen#10]
Hello,

Each time I try to load the maqueen (v 1.2.1) extension into makecode, I have:

```
SyntaxError: JSON.parse: expected double-quoted property name at line 25 column 1 of the JSON data
2 pxtapp.js:9159:26
```

On OSX, whatever the browser (safari, chrome or firefox)

**Comment by tangjie133** [comment]
I tested but no problem!

**Comment by nsteinmetz** [comment]
What is weird is that:
* If I start an empty project, load the extension it fails
* If I open a project which assumes having the extensions, it works - that's the way I could use the extension but I don't know which version I use then...

**Comment by tangjie133** [comment]
Does this problem still exist?

**Comment by nsteinmetz** [comment]
Yes :

![Capture d’écran du 2020-01-14 10-55-09](https://user-images.githubusercontent.com/33180/72333652-6b4a3b80-36bc-11ea-8902-3ed9e15b26e3.png)

**Comment by tangjie133** [comment]
Where does your MaQueen come from, is it from the official website, can you give me a big screenshot, full screen?

**Comment by nsteinmetz** [comment]
Hi,

What I do:
* Go to https://makecode.microbit.org/#editor
* Click on the wheel (top right corner) > Extensions > Look for maqueen in the search engine
* Click on maqueen (if you click on dislay more, it shows 1.2.1 version) and refer to this page : https://makecode.microbit.org/pkg/dfrobot/pxt-maqueen
* Then, it goes back to the editor and loading mechanism is done indefinitely and I have these errors in the console

![Capture d’écran du 2020-01-15 09-19-49](https://user-images.githubusercontent.com/33180/72417034-461b0300-3778-11ea-8382-95215df1006f.png)

Thanks for trying to understand what happens and why it does not work. Really appreciated !

**Comment by tangjie133** [comment]
Can you use other libraries from the official website？

**Comment by nsteinmetz** [comment]
Yes, I could load bluetooth, bitbot and kitronic-servo-lite without any issue.

**Comment by patrickpollet1966** [comment]
Hi,

I just tried to work with a maqueen with my son and i have exactly the same problem as nsteinmetz.

(i tried with Chrome and Safari under OSX)

![image](https://user-images.githubusercontent.com/12225432/72686009-58869b00-3af0-11ea-8891-73e8d24e320e.png)

**Comment by erwangrouj** [comment]
I also have the problem if I use the french language. Switching Makecode to english language makes it work fine.

The problem comes from the ending comma in the french translation file (__locales/fr/maqueen-strings.json)

forking the maqueen repo then removing the ending comma made it work in french for me :

![maqueen_locale_fr](https://user-images.githubusercontent.com/60329639/73484530-c741df00-43a1-11ea-93b2-ce0ab2e5d46f.JPG)

**Comment by nsteinmetz** [comment]
@erwangrouj brilliant, I was thinking about it the other day but did not take time to check this. Was more thinking about a bad escaped `'` but comma works also ;-)

**Comment by nsteinmetz** [comment]
@erwangrouj why not submit a pull request then ?
